src/http_module.cpp

```diff
@@ -513,19 +502,6 @@ void grpcLogHandler(gpr_log_func_args* args)
     ngx_log_error(level, ngx_cycle->log, 0, "OTel/grpc: %s", args->message);
```

Comment:
Let's move all the gRPC/Protobuf log stuff into dedicated .cpp file (e.g. grpc_log.cpp). The corresponding header file should probably expose only one function (e.g. initGrpcLog).

Reply:
See [ccb30e1](https://github.com/nginxinc/nginx-otel/pull/34/commits/ccb30e11659828c2fb9efa0229cecee0cdcb4237).